power_grid_model_c/power_grid_model/include/power_grid_model/component/transformer.hpp

```diff
@@ -23,7 +23,7 @@ class Transformer : public Branch {
     using UpdateType = TransformerUpdate;
     static constexpr char const* name = "transformer";
 
-    explicit Transformer(TransformerInput const& transformer_input, double u1_rated, double u2_rated)
+    explicit Transformer(TransformerInput const& transformer_input, double u1_rated = 1.0, double u2_rated = 1.0)
```

Comment:
why? this is different from what we currently do in the rest of the PGM so we should consider this carefully. if you really need this, please add an explicit overload

```suggestion
    explicit Transformer(TransformerInput const& transformer_input): Transformer{.transformer_input = transformer_input, .u1_rated = 1.0, .u2_rated = 1.0} {};
    // or just explicit Transformer(TransformerInput const& transformer_input): Transformer{.transformer_input, 1.0, 1.0} {};
    explicit Transformer(TransformerInput const& transformer_input, double u1_rated, double u2_rated)
```

or if that doesn't compile just
```suggestion
    explicit Transformer(TransformerInput const& transformer_input): Transformer{.transformer_input, 1.0, 1.0} {};
    explicit Transformer(TransformerInput const& transformer_input, double u1_rated, double u2_rated)
```

Reply:
See https://github.com/PowerGridModel/power-grid-model/issues/574 
Revert is simple so long as it doesn't impose any issues.